docs/firmware/zephyr-device-sdk/light-db/README.md

```diff
@@ -0,0 +1,24 @@
+---
+title: LightDB Client Overview
+sidebar_position: 1
+---
+
+LightDB is a database service hosted by Golioth. It can be used to create applications involving resource state. Device state such as LED on/off, door lock position, or thermostat settings are easy to manage using the Golioth cloud.
+
+The key differentiator between the LightDB and LightDB Stream services offered by Golioth is data persistence. LightDB data values do not accumulate, they are replaced with update calls. The LightDB Stream service can be used to create time series data.
```

Comment:
As another differentiator, it might be worth noting that LightDB stream is unidirectional (device to cloud), whereas LightDB state is bi-directional.

Reply:
This is great point. I've tried to encapsulate it in this edit: 

```
The key differentiator between the LightDB State and LightDB Stream services
offered by Golioth is data persistence. LightDB data values do not accumulate,
they are replaced with update calls. LightDB State is bidirectional; changes may
originate from either the device or the cloud (e.g. via an API call or changes
made on the Golioth Web Console). LightDB Stream is a unidirectional service
(device to cloud) useful for recording time series data.
```